conf/supportability.yml

```diff
@@ -0,0 +1,4 @@
+supportability:
```

Comment:
Why we are having `supportability` YAML instead of being specific for RHEL Versions or just versions (versions.yaml)? What else you are planning to add to this supportability.yaml ?

If not, even this could have been a part of existing content_host.yaml with just one line under content_host header/context:

```
versions: [6, 7, 8, 9]
```

Reply:
I think one of the goals here is to have the supportability isolated from the rest of the configs, since the way we handle the configs now is that we share them across all robottelo branches (satellite) versions.
However, you can't have rhel9 support in older sat version support matrices.
Also, we agreed that handling this from within a pipeline (by injecting dynaconf vars based on some conditionals) is not really well maintainable, nor readable).

Reply:
Supportability can also include things like what versions of hypervisors Satellite supports, but is separately defining what Satellite supports. Not necessarily what will be tested.

This allows for clear logical separation during test generation and collection between what Sat supports and what we want to generate for.

If I just state it as a list of versions in content_host, its more difficult to grow and scale as we do things like support hosts other than RHEL.  It also doesn't provide the logical separation.

Reply:
@mshriver Satellite really doesn't provisions/differs functionality based on hypervisor versions! Also, Satellite doesn't supports hosts other than RHEL(as you stated above). I commented here because I dont recall anything apart from RHEL versions that need to sit under the `supportability` matrix! Even in CI under `PipelineVars.Groovy` we have a RHEL_Versions as the only supportability matrix there! The max I can think of is Satellite versions apart from RHEL Versions !

But agree that this provides a clear separation between test generation and collection !